layers/utils.py

```diff
@@ -27,10 +27,21 @@ def convertPbfToGeoJson(pbfPath, geojsonPath, geometryType):
     )
 
 
-# Note the layer name is based on the input filename. This always generates
+# Note the layer name is based on the output filename. This always generates
 # numeric feature IDs.
 def convertGeoJsonToPmtiles(geojsonPath, pmtilesPath):
-    run(["tippecanoe", geojsonPath, "--generate-ids", "-o", pmtilesPath])
+    layerName = os.path.basename(pmtilesPath)[: -len(".pmtiles")]
```

Comment:
This is less error prone; we always name the final output how we want, but sometimes the input could be "extract.geojson" or something else